bin/node/cli/src/service.rs

```diff
@@ -99,6 +99,7 @@ macro_rules! new_full_start {
 					Some(Box::new(justification_import)),
 					None,
 					client,
+					select_chain,
```

Comment:
@andresilva Do light clients _need_ a select chain instance? Is this something we could turn into an `Option`?

Reply:
I think this can be turned into an option. This was added to fetch some data required for submitting equivocation reports, which the light clients won't do for now as only block authors submit them.